src/main/java/org/checkerframework/specimin/TargetMethodFinderVisitor.java

```diff
@@ -27,11 +27,7 @@
 import com.github.javaparser.resolution.types.ResolvedReferenceType;
 import com.github.javaparser.resolution.types.ResolvedType;
 import com.google.common.base.Splitter;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
Please avoid wildcard imports (our house style is no wildcard imports)

Reply:
thanks for letting me know - avoiding moving forward